File: MovieBooking/Services/Reservations/Reservations.API/Entities/Reservation.cs
using System;

namespace Reservations.API.Entities
{
    public class Reservation
    {
        public string ProjectionId { get; set; }
        public string ProjectionDate { get; set; }
        public string ProjectionTerm { get; set; }
        public string MovieId { get; set; } 
        public string MovieTitle { get; set; }
        public string TheaterHallId { get; set; }
        public string TheaterHallName { get; set; }
        public int Price { get; set; }
        public int NumberOfTickets { get; set; }
    }
}

File: MovieBooking/MovieBookingSPA/src/app/administration/domain/models/reservation.ts
export interface IReservation{
    id:number,
    projectionId: string;
    projectionDate: string;
    projectionTerm: string;
    movieId: string;
    movieTitle: string;
    theaterHallName: string;
    theaterHallId: string;
    price: number;
    numberOfTickets: number;
}